testing/gimmeproj/main.go

```diff
@@ -125,12 +126,19 @@ Administrative commands:
 		return fmt.Errorf("datastore.NewClient: %v", err)
 	}
 
+	// List is intended explicitly for scripting.
+	if flag.Arg(0) != "list" {
+		fmt.Println(flag.Arg(0), flag.Arg(1))
+	}
+
 	switch flag.Arg(0) {
 	case "help":
```

Comment:
add list here.

if facilitating scripting, specifically, add a `-json` or `-output=json` flag.

would adding that to "status" work fine?